packages/build/webpack/package.config.js

```diff
@@ -114,8 +109,6 @@ module.exports = {
         'jest-environment-jsdom',
         'jquery',
         'lodash',
```

Comment:
The package `date-fns` is already listed here. I attempted to add `date-fns-tz` as well, however, that ended up causing errors in compilation for our tests. Could consider adding it back and triaging but currently just going to leave as-is.